recipes/lagrange-ng/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export MKLROOT=${PREFIX}
+export CMAKE_PREFIX_PATH=${PREFIX}
+
+cmake ${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -Bbuild -H.
+pushd build
+make
+popd
+
```

Comment:
Is there a `make check` or `ctest` we can run here?

Reply:
there is a `tests` recipe in the makefile, but it will download gtest and build the entire test suite. Should I add a command to check if the software has been built properly?

Reply:
We have `gtest` in conda-forge. Can you try depending on it for the build/tests?